extract_endpoint/src/extract_endpoint/endpoint.py

```diff
@@ -63,6 +63,8 @@ def upload_file() -> typing.Tuple[str, int]:
         flask.abort(HTTPStatus.BAD_REQUEST)
     if 'file' not in flask.request.files:
         flask.abort(HTTPStatus.BAD_REQUEST)
+    if flask.request.form.get('timestamp', None) == '':
```

Comment:
should compare to `None` rather than ''

Reply:
Also no need for the `None` in the `get` call, that's the default value anyway.